packages/plugin-commands-installation/src/recursive.ts

```diff
@@ -163,6 +164,9 @@ export default async function recursive (
     delete opts.include
   }
 
+  const patternedInput = input.filter(i => i.includes('*'))
```

Comment:
to avoid filtering again in `matchDependencies()`. You may use `R.partial` and get two arrays. One with the patternedInput and another with the exact dependnency names.

Reply:
That's neat!

Reply:
Sorry, the right method is https://ramdajs.com/docs/#partition